uw-frame-components/portal/widgets/partials/compact-widget-card.html

```diff
@@ -0,0 +1,24 @@
+<md-card class="list-content" id="widget-id-{{::widget.nodeId}}">
+  <md-button class="widget-action widget-info md-icon-button" aria-label="see a brief description of this app">
+    <md-tooltip md-direction="top" class="widget-action-tooltip">
+      {{widget.description}}
+    </md-tooltip>
+    <md-icon>info</md-icon>
+  </md-button>
+  <div ng-transclude id="widget-removal"></div>
+<!--   <md-button class="widget-action widget-remove md-icon-button" -->
+<!--              aria-label="remove {{ widget.title }} widget from your home screen" -->
+<!--              ng-click="layoutCtrl.removePortlet(widget.nodeId, widget.title)" -->
+<!--              ng-hide="GuestMode || cantRemove"> -->
+<!--     <md-icon>close</md-icon> -->
+<!--   </md-button> -->
```

Comment:
Great catch!  Removed it to prove that I didn't need it, but didn't actually remove it